services/users/user.go

```diff
@@ -0,0 +1,128 @@
+	IsSuperUser           bool                  `json:"isSuperUser"`
+	IsAuditor             bool                  `json:"isAuditor"`
+	HasAPIKey             bool                  `json:"hasApiKey"`
+	HasAPIKeyV1           bool                  `json:"hasApiKeyV1"`
+	HasAPIKeyV2           bool                  `json:"hasApiKeyV2"`
+	IsMfaEnabled          bool                  `json:"isMfaEnabled"`
+	SsoEnabled            bool                  `json:"ssoEnabled"`
+	RoleIds               []int                 `json:"roleIds"`
+	IamSafe               IamSafe               `json:"iamSafe"`
+	CanSwitchRole         bool                  `json:"canSwitchRole"`
+	IsLocked              bool                  `json:"isLocked"`
+	LastLogin             time.Time             `json:"lastLogin"`
+	Permissions           Permissions           `json:"permissions"`
+	CalculatedPermissions CalculatedPermissions `json:"calculatedPermissions"`
+	IsMobileDevicePaired  bool                  `json:"isMobileDevicePaired"`
+}
+type IamEntitiesLastLeaseTime struct {
+	IamEntity     string    `json:"iamEntity"`
+	LastLeaseTime time.Time `json:"lastLeaseTime"`
+}
+type CloudAccounts struct {
+	CloudAccountID           string                     `json:"cloudAccountId"`
+	Name                     string                     `json:"name"`
+	ExternalAccountNumber    string                     `json:"externalAccountNumber"`
+	LastLeaseTime            time.Time                  `json:"lastLeaseTime"`
+	State                    string                     `json:"state"`
+	IamEntities              []string                   `json:"iamEntities"`
+	IamEntitiesLastLeaseTime []IamEntitiesLastLeaseTime `json:"iamEntitiesLastLeaseTime"`
+	CloudAccountState        string                     `json:"cloudAccountState"`
+	IamEntity                string                     `json:"iamEntity"`
+}
+type IamSafe struct {
+	CloudAccounts []CloudAccounts `json:"cloudAccounts"`
+}
+type Permissions struct {
+	Access             []string `json:"access"`
+	Manage             []string `json:"manage"`
+	Rulesets           []string `json:"rulesets"`
+	Notifications      []string `json:"notifications"`
+	Policies           []string `json:"policies"`
+	AlertActions       []string `json:"alertActions"`
+	Create             []string `json:"create"`
+	View               []string `json:"view"`
+	CrossAccountAccess []string `json:"crossAccountAccess"`
+}
+type CalculatedPermissions struct {
+	Access             []string `json:"access"`
+	Manage             []string `json:"manage"`
+	Rulesets           []string `json:"rulesets"`
+	Notifications      []string `json:"notifications"`
+	Policies           []string `json:"policies"`
+	AlertActions       []string `json:"alertActions"`
+	Create             []string `json:"create"`
+	View               []string `json:"view"`
+	CrossAccountAccess []string `json:"crossAccountAccess"`
+}
+
+func (service *Service) Get(userId int64) (*UserResponse, *http.Response, error) {
```

Comment:
So far we received IDs as strings (in opposite to the api). Let's be consistent about this